pkg/client/client_test.go

```diff
@@ -1430,6 +1430,130 @@ var _ = Describe("Client", func() {
 				close(done)
 			}, serverSideTimeoutSeconds)
 
+			It("should filter results using limit and continue options", func(done Done) {
+				By("creating 4 deployments")
+				dep1 := &appsv1.Deployment{
+					ObjectMeta: metav1.ObjectMeta{
+						Name: "deployment-1",
+					},
+					Spec: appsv1.DeploymentSpec{
+						Selector: &metav1.LabelSelector{
+							MatchLabels: map[string]string{"foo": "bar"},
+						},
+						Template: corev1.PodTemplateSpec{
+							ObjectMeta: metav1.ObjectMeta{Labels: map[string]string{"foo": "bar"}},
+							Spec:       corev1.PodSpec{Containers: []corev1.Container{{Name: "nginx", Image: "nginx"}}},
+						},
+					},
+				}
+				dep1, err := clientset.AppsV1().Deployments(ns).Create(dep1)
+				Expect(err).NotTo(HaveOccurred())
+				dep2 := &appsv1.Deployment{
```

Comment:
refactor this out into a little helper.  It can live in this test, if you want (e.g. `makeDeployment := func(suffix string) *appsv1.Deployment { ... }`)